keyboards/pimoroni/keybow2040/config.h

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET
+
+#define RGB_MATRIX_LED_COUNT (4 * 4)
+#define RGB_MATRIX_KEYPRESSES
+
+/*
+ * LED driver chip count: one IS31FL3731
+ */
+#define DRIVER_COUNT 1
+
+/*
+ * I2C address: IS31FL3731 AD pin is connected to ground
+ */
+#define DRIVER_ADDR_1 0b1110100
+
+/*
+ * The RP2040 has two I2C peripherals; I2C0 must be used with GPIO 4
+ * and 5 which on the Keybow 2040 are wired up to the IS31FL3731.
+ */
+#define I2C_DRIVER I2CD0
+
+/*
+ * I2C1 here refers to the board's / QMK's first (and only) I2C
+ * peripheral, not to the RP2040's second I2C peripheral.
+ */
```

Comment:
```suggestion
#define IS31FL3731_I2C_ADDRESS_1 IS31FL3731_I2C_ADDRESS_GND

#define I2C_DRIVER I2CD0
```

Reply:
ISSI addresses should now use their respective defines instead of literals, for consistency and clarity.

Reply:
Because things were refactored last develop cycle?

Reply:
OK, I see `#define DRIVER_ADDR_1` needs changing since d56ee70c524b4fc4d1638e5e8c4bdeb89e752993

Other than that, it's wrong to remove the comments since they explain where the values come from.